locations/spiders/franco_manca_gb.py

```diff
@@ -1,17 +1,17 @@
 from scrapy.spiders import SitemapSpider
 
-from locations.open_graph_parser import OpenGraphParser
+from locations.open_graph_spider import OpenGraphSpider
 from locations.pipelines.address_clean_up import merge_address_lines
 from locations.structured_data_spider import extract_phone
 
 
-class FrancoMancaGBSpider(SitemapSpider):
+class FrancoMancaGBSpider(SitemapSpider, OpenGraphSpider):
     name = "franco_manca_gb"
     item_attributes = {"brand": "Franco Manca", "brand_wikidata": "Q28404417"}
     sitemap_urls = ["https://www.francomanca.co.uk/restaurant-sitemap.xml"]
+    wanted_types = ["article"]
```

Comment:
To be checked, this dropped from 44 items to 1 in CI.
I checked one page was an article; need to confirm more